doc/build-instructions/Build_Instructions_V25.md

```diff
@@ -0,0 +1,802 @@
+
+Download the docker build script to your local system or copy and paste the following command:
+
+```
+wget https://raw.githubusercontent.com/eclipse-openj9/openj9/master/buildenv/docker/mkdocker.sh
+```
+
+Optionally, skip to [Setting up your build environment without Docker](#setting-up-your-build-environment-without-docker).
+
+#### Setting up your build environment with Docker :whale:
+If you want to build a binary by using a Docker container, follow these steps to prepare your system:
+
+1. The first thing you need to do is install Docker. You can download the free Community edition from [here](https://docs.docker.com/engine/installation/), which also contains instructions for installing Docker on your system.  You should also read the [Getting started](https://docs.docker.com/get-started/) guide to familiarise yourself with the basic Docker concepts and terminology.
+
+2. Next, run the following command to build a Docker image, called **openj9**:
+```
+bash mkdocker.sh --tag=openj9 --dist=ubuntu --version=22 --gitcache=no --jdk=25 --build
+```
+
+3. Start a Docker container from the **openj9** image with the following command, where `-v` maps any directory, `<host_directory>`,
+on your local system to the containers `/root/hostdir` directory so that you can store the binaries, once they are built:
+```
+docker run -v <host_directory>:/root/hostdir -it openj9
+```
+
+:pencil: Depending on your [Docker system configuration](https://docs.docker.com/engine/reference/commandline/cli/#description), you might need to prefix the `docker` commands with `sudo`.
+
+Now that you have the Docker image running, you are ready to move to the next step, [Get the source](#2-get-the-source).
+
+#### Setting up your build environment without Docker
+
+If you don't want to user Docker, you can still build directly on your Ubuntu system or in a Ubuntu virtual machine. Use the output of the following command like a recipe card to determine the software dependencies that must be installed on the system, plus a few configuration steps.
+
+```
+bash mkdocker.sh --tag=openj9 --dist=ubuntu --version=22 --gitcache=no --jdk=25 --print
+```
+
+1. Install the list of dependencies that can be obtained with the `apt-get` command from the following section of the Dockerfile:
+```
+apt-get update \
+  && apt-get install -qq -y --no-install-recommends \
+    ...
+```
+
+2. The previous step installed g++-11 and gcc-11 packages, which might be different
+than the default version installed on your system. Export variables to set the
+version used in the build.
+```
+export CC=gcc-11 CXX=g++-11
+```
+
+3. Download and setup the boot JDK using the latest AdoptOpenJDK v25 build.
+```
+cd <my_home_dir>
+wget -O bootjdk25.tar.gz "https://api.adoptopenjdk.net/v3/binary/latest/25/ga/linux/x64/jdk/openj9/normal/adoptopenjdk"
+tar -xzf bootjdk25.tar.gz
+rm -f bootjdk25.tar.gz
+mv $(ls | grep -i jdk-25) bootjdk25
```

Comment:
Java 25 is not available via that URL as I write this. Java 24 can be used to boot jdk25 so we could use that instead, or do you think Java 25 will be available soon enough?

Reply:
That seems reasonable.